### PR TITLE
documentation for force link to social idp

### PR DESCRIPTION
**Pull request by royalchan-okta**
@larsjohansen-okta  @richardmateosian-okta

### CODE REVIEW

**Comment by larsjohansen-okta** [comment]
There's nowhere in the docs we specify that the idp has to be a social provider, so I bet we'd have customers wondering why they get a RNF exception when they provide a "valid" input in their mind. Maybe we could mention this feature only works for social idps in the doc?

**Comment by royalchan-okta** [comment]
Added that with the url path switches

File: _source/_docs/api/resources/idps.md [hunk 1]
```diff
@@ -2905,6 +2905,219 @@ curl -v -X GET \
 }
 ~~~
 
+## Identity Provider User Operations
+
+### Listing IdPs associated with a user
+{:.api .api-operation}
+
+<span class="api-uri-template api-uri-get"><span class="api-label">GET</span> /api/v1/users/*:uid*/idps</span>
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/idps.md]: Can we use the new-style {% api_operation GET /api/v1/users/:uid/idps %} ?

File: _source/_docs/api/resources/idps.md [hunk 2]
```diff
@@ -2905,6 +2905,219 @@ curl -v -X GET \
...
+  "errorLink": "E0000007",
+  "errorId": "oaeYW9k9yJuSSSkhaMQdA1-Zg",
+  "errorCauses": []
+}
+~~~
+
+### Link a user to a Social IdP without a transaction
+{:.api .api-operation}
+
+<span class="api-uri-template api-uri-post"><span class="api-label">POST</span> /api/v1/idps/*:id*/users/*:uid*</span>
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/idps.md]: See comment for 2913.

File: _source/_docs/api/resources/idps.md [hunk 3]
```diff
@@ -2905,6 +2905,219 @@ curl -v -X GET \
...
+
+Links an Okta User to an existing Social [Identity Provider](#identity-provider-model). This does not support the SAML2 [Identity Provider Type](#identity-provider-type).
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/idps.md]: "Okta user" is typically how we write this.
[Social Identity Provider] or if that's not a feature name, social [Identity Provider]

**Pull request closed without merging**